modules/i3s/docs/api-reference/i3s-loader.md

```diff
@@ -168,6 +168,7 @@ const visibleTiles = tileset3d.tiles.filter(tile => tile.selected);
 | `options.i3s.tile`                  | `Object`         | `null`  | `Tile` object loaded by I3SLoader or follow the data format [Tile Object](#tile-object). It is required when loading i3s geometry content                                                                                                                                                                                       |
 | `options.i3s.useDracoGeometry`      | `Bool`           | `true`  | Use 'Draco' compressed geometry to show if applicable                                                                                                                                                                                                                                                                           |
 | `options.i3s.useCompressedTextures` | `Bool`           | `true`  | Use "Compressed textures" (_.dds or _.ktx) if available and supported by GPU                                                                                                                                                                                                                                                    |
+| `options.i3s.useFeatureIds`         | `Bool`           | `true`  | Add `featureIds` property in tile content if available in geometry. `featureIds` specified in [tiles-3d-loader](https://loaders.gl/modules/3d-tiles/docs/api-reference/tiles-3d-loader) .                                                                                                                                       |
```

Comment:
We should specify this in the `tiles` module (the common format for I3S and 3D Tiles) and link to that. We should not cross reference I3S and 3D Tiles.

Please land https://github.com/visgl/loaders.gl/pull/1408 first and then rebase and update this.

Reply:
Ok, will start from #1408